Failed to load plugin react: Cannot find module 'eslint-plugin-react' if use standardjs

Why standarjs is looking for react just wanated to setup simple express nodejs. 

$ yarn start
yarn run v1.3.2
$ neutrino start
× Building project failed
./src/index.js
Module build failed: Error: Failed to load plugin react: Cannot find module 'eslint-plugin-react'
Referenced from:
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.resolve (internal/module.js:18:19)
    at Plugins.load (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\plugins.js:106:29)
    at Array.forEach (<anonymous>)
    at Plugins.loadAll (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\plugins.js:146:21)
    at loadFromDisk (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\config-file.js:492:35)
    at load (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\config-file.js:550:20)
    at configExtends.reduceRight (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\config-file.js:421:36)
    at Array.reduceRight (<anonymous>)
    at applyExtends (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\config-file.js:403:28)
    at Object.loadObject (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config\config-file.js:529:35)
    at new Config (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\config.js:72:46)
    at new CLIEngine (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint\lib\cli-engine.js:420:23)
    at Object.module.exports (C:\Data\SVD_Work\Projects\BI_Monitoring_2\node_modules\eslint-loader\index.js:176:27)
 @ multi ./src/index (webpack)/hot/poll.js?1000
error Command failed with exit code 1.


configuration of packages.js
{
  "version": "1.0.0",
  "private": true,
  "description": "Get started with Express!",
  "main": "index.js",
  "engines": {
    "node": ">=6.9"
  },
  "scripts": {
    "start": "neutrino start",
    "build": "neutrino build"
  },
  "author": "Hassan Ali <helfi92@gmail.com>",
  "license": "MIT",
  "devDependencies": {
    "@neutrinojs/node": "^8.0.15",
    "@neutrinojs/standardjs": "^8.0.15",
    "neutrino": "^8.0.15"
  },
  "dependencies": {
    "express": "^4.15.3"
  }
}

neutrino setup. 
module.exports = {
  use: [
    '@neutrinojs/node',
    '@neutrinojs/standardjs'
  ]
};

--- a/packages/standardjs/package.json
+++ b/packages/standardjs/package.json
@@ -31,6 +31,7 @@
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-node": "^5.2.1",
     "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-react": "^7.5.1",
     "eslint-plugin-standard": "^3.0.1"
   },
   "peerDependencies": {

